test/test-script.js

```diff
@@ -546,63 +546,58 @@ describe('RangedZombie', function() {
   });
 
   it('should have health', function() {
-    var spitter = new RangedZombie(30, 15, 15);
     spitter.health.should.equal(30);
   });
 
   it('should have a strength attribute', function() {
-    var spitter = new RangedZombie(30, 15, 15);
     spitter.strength.should.equal(15);
   });
 
   it('should have a speed attribute', function() {
-    var spitter = new RangedZombie(30, 15, 15);
     spitter.speed.should.equal(15);
   });
 
   it('should have a private maxHealth variable', function() {
-    var spitter = new RangedZombie(30, 15, 15);
     expect(spitter.maxHealth).to.be.undefined;
   });
 
   it('should be alive', function() {
-    var spitter = new RangedZombie(30, 15, 15);
     spitter.isAlive.should.equal(true);
   });
 
   it('should have a range attribute', function() {
-    var spitter = new RangedZombie(30, 15, 15);
     spitter.range.should.equal(225);
-  })
+  });
 
   it('should have a puke method', function() {
-    var spitter = new RangedZombie(30, 15, 15);
     spitter.puke.should.exist;
-  })
+  });
 
   it('should print out a message including the range it pukes when zombie pukes', function() {
     sandbox.stub(console, "log");
-    var spitter = new RangedZombie(30, 15, 15);
     spitter.puke();
     sinon.assert.called(console.log);
-  })
+  });
 
   it('should decrease health by 1 everytime zombie pukes', function() {
-    var spitter = new RangedZombie(30, 15, 15);
     spitter.puke();
     spitter.puke();
     spitter.health.should.equal(28);
-  })
+  });
 
 }); // end RangedZombie specs
 
+var boomer;
```

Comment:
`var boomer`; should go inside `describe('ExplodingZombie'...`
just before `beforeEach`